### DISCUSSION TITLE
Specific rollover time (not midnight) while configuring daily rollover in TimeBasedRollingPolicy

### DISCUSSION THREAD

**Discussion by randrzej** [qos-ch/logback#644] [Q&A]
Hi,

Based on the docs it seems when specifying daily rollover, there is no way to tell at what time rollover should be done (it happens at midnight).

While I understand the reasoning, is there a way to change this so that the rollover happens every day at noon? If so, what's the easiest way to accomplish this?

Thank you in advance for your help.

Regards,
Remi

**Comment by ceki** [comment]
The current code performs rollover upon change of the current period. There is no way to specify a different the rollover instant. However, you can adapt existing code to suit your needs.

**Reply by randrzej** [reply]
I was thinking about using TimeZone parameter to shift the rollover e.g. GMT-12. This would be a workaround, but it would achieve my goal, which is not to rollover while the nightly job is running (the job gets triggered at 20:00 and runs for 8h).
